consensus/Cargo.toml

```diff
@@ -30,4 +29,3 @@ tokio = { version = "1", features = ["full"] }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 wasm-timer = "0.2.5"
```

Comment:
We'll need to replace usage of wasm-timer with a call to `ic_cdk::api::time()` or the `ic0` crate.
We can do that in next PR's, if so please add todo

Reply:
also just to note, wasm-timer is abandoned and author is against using it https://github.com/tomaka/wasm-timer/issues/21#issuecomment-1380544611